index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Review code</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Hello World ! , We are Software developers</h1>
+    <p id="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
+
```

Comment:
You have done a good job with your code 👍 but Please, try to cut down on the number of empty lines in your code. Less empty space can result in faster page loads and a better user experience. Empty lines in HTML code can be removed if they are unneeded. Additionally, it makes the code shorter and simpler to understand, which is advantageous for engineers working on the code.